config/delta_service.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Database Connection
+    |--------------------------------------------------------------------------
+    */
+    'connection' => [
```

Comment:
omdat de package anders niet gepublished kan worden. De uiteindelijke concrete config waarden staan nogsteeds in .env check de config map daar staat de gepublishte config file op basis van deze abstractie